app/ui/src/app/integrations/edit-page/step-configure/data-mapper/data-mapper-host.component.ts

```diff
@@ -43,7 +43,7 @@ const MAPPING_KEY = 'atlasmapping';
 @Component({
   selector: 'syndesis-data-mapper-host',
   template: `
-    <div *ngIf="initialized" class="data-mapper-host">
+    <div *ngIf="initializedCountDown==0" class="data-mapper-host">
```

Comment:
Spacing out the equality expression like this will enhance readability:

`<div *ngIf="initializedCountDown == 0" class="data-mapper-host">`